tests/mmm/test_delayed_saturated_mmm.py

```diff
@@ -872,27 +872,39 @@ def test_plot_new_spend_contributions_prior_select_channels(
 @pytest.fixture(scope="module")
 def fixed_model_parameters() -> dict[str, Union[float, list[float]]]:
     return {
-        "intercept": 2.5,
-        "beta_channel": [0.5, 0.5],
+        "intercept": 5.0,
+        "beta_channel": [0.15, 0.5],
         "alpha": [0.5, 0.5],
         "lam": [0.5, 0.5],
         "likelihood_sigma": 0.25,
-        "gamma_control": [0.5, 0.5],
+        "gamma_control": [0.0001, 0.005],
     }
 
 
+def random_mask(df: pd.DataFrame, mask_value: float = 0.0) -> pd.DataFrame:
+    shape = df.shape
+
+    mask = rng.choice([0, 1], size=shape, p=[0.75, 0.25])
+    return df.mul(mask)
+
+
+@pytest.fixture(scope="module")
+def masked_toy_X(toy_X) -> pd.DataFrame:
+    return toy_X.set_index("date").pipe(random_mask).reset_index()
```

Comment:
The rational here is that since all of the toy_X columns are pretty much white noise, the generated y is also pretty much a white noise and the model doesn't fit well

Here there are some clear stops in toy_X making the y notably depend on the covariates